Makefile.am

```diff
@@ -81,6 +81,8 @@ all-local: digraphs.la
 	cp -RL @PLANARITY_SUITE_DIR@/.libs/*  $(top_srcdir)/bin/lib/
 if SYS_IS_CYGWIN
 	cp .libs/digraphs.dll $(GAPINSTALLLIB)
+# Cygwin will only look in this directory for dlls
+	cp bin/lib/cygplanarity-0.dll $(GAPROOT)/.libs
```

Comment:
I wouldn't do this.   In general it doesn't make sense to be manually copying anything into `.libs`.  This happens to work, because when you run `./gap` out of a source build you're actually running the [wrapper executable](https://www.gnu.org/software/libtool/manual/html_node/Wrapper-executables.html) generated by libtool, which in turns runs the actual `gap.exe` that is under `.libs`.  

Thus, this happens to work, since the planarity.dll used by the package is now under the gap.exe executable's DLL search path.  

I'll take a closer look at this to make sure my suggestion works, but I think a better approach would be to "install" any DLLs to `$(abs_top_srcdir/$(BINARCHDIR)`.

Reply:
Looking at windows, the places cygwin tries to find this DLL is:

/bin, /lib, /usr/bin and friends (don't really want to start putting things there)

The only places it looks inside the GAP directory are:

the gap root directory and the subdirectories .libs and gap/extern/install/zlib/zlib/lib